### ISSUE TITLE
Having trouble reproducing paper results with provided checkpoints

### ISSUE THREAD

**Issue by LunjunZhang** [tarashakhurana/4d-occ-forecasting#8]
Hi there,

I downloaded the NuScenes checkpoints (for both 1s and 3s prediction) and ran the evaluation code for the checkpoints. I am having trouble reproducing the reported results in the paper from the checkpoints. More specifically, the _reported_ / **reproduced** results are:

 - 1s prediction: 
   - L1: _1.40_ / **1.463**
   - AbsRel: _10.37_ / **10.97**
   - Chamfer Distance Near-Field: _1.41_ / **1.4859**
   - Chamfer Distance Vanilla: _2.81_ / **2.9902**

- 3s prediction:
   - L1: _1.71_ / **1.903**
   - AbsRel: _13.48_ / **15.54**
   - Chamfer Distance Near-Field: _1.40_ / **1.7462**
   - Chamfer Distance Vanilla: _4.31_ / **5.2942**

Thanks for your help!

**Comment by tarashakhurana** [comment]
Hi, the results we report are on the validation set of NuScenes. Is that what you tested the checkpoints on?

**Comment by LunjunZhang** [comment]
Thanks a lot for the reply! 

We tested the checkpoint on the test set of NuScenes, which I think is the default behaviour of `test.py` in the repo. We will try changing the evaluation dataset to the validation set instead. 

Are the numbers for other datasets all done on the validation set (rather than the test set) as well?

**Comment by tarashakhurana** [comment]
Sorry for the confusion! For nuScenes, the only baselines we had access to (SPF and S2Net) were evaluated on the validation set. Their paper says test set but we confirmed this with the authors. All other datasets should be test sets.

Edit: Another reason for using the validation set for nuScenes was that the LidarSeg labels were not available for the test set, which we used to compute the foreground/background metrics.

Edit 2: Just to be clear, `test.py` does not have this default behavior. `test.sh` lists the test set for KITTI. `test_fgbg.sh` lists the val set for nuScenes. Let me know if the metrics still don't match.
